java/component/keyple-core/src/main/java/org/eclipse/keyple/core/service/util/ContactsCardCommonProtocols.java

```diff
@@ -17,11 +17,11 @@
  * @since 1.0
  */
 public enum ContactsCardCommonProtocols {
```

Comment:
Rename ContactsCardCommonProtocols to ContactCardCommonProtocols

Reply:
Yes.